src/main/java/Task/Task.java

```diff
@@ -0,0 +1,73 @@
+package Task;
+
+public class Task {
+    public enum TaskType {
+        T,
+        D,
+        E,
+    }
+    private String taskName;
+    private boolean completed;
```

Comment:
Might want to use the prefix "is" in the name of this field e.g isCompleted for better readability